topics/sequence-analysis/tutorials/quality-control/tutorial.md

```diff
@@ -871,10 +877,224 @@ These datasets can be used for the downstream analysis, e.g. mapping.
 > {: .solution}
 {: .question}
 
+# Assess quality with Nanoplot - Long reads only
+
+In case of long reads, we can check sequence quality with [Nanoplot](https://github.com/wdecoster/NanoPlot/) {% cite 10.1093/bioinformatics/bty149 %}. It provides basic statistics with nice plots for a fast quality control overview.
```

Comment:
```suggestion
In case of long reads, we can check sequence quality with [Nanoplot](https://github.com/wdecoster/NanoPlot/) ({% cite 10.1093/bioinformatics/bty149 %}). It provides basic statistics with nice plots for a fast quality control overview.
```